src/Microsoft.TestPlatform.ObjectModel/Navigation/DiaSession.cs

```diff
@@ -101,12 +101,21 @@ private static ISymbolReader GetSymbolReader(string binaryPath)
         // For remote scenario, also look for pdb in current directory, (esp for UWP)
         // The alternate search path should be an input from Adapters, but since it is not so currently adding a HACK
         pdbFilePath = !File.Exists(pdbFilePath) ? Path.Combine(Directory.GetCurrentDirectory(), Path.GetFileName(pdbFilePath)) : pdbFilePath;
-        using var stream = new FileHelper().GetStream(pdbFilePath, FileMode.Open, FileAccess.Read);
-        if (PortablePdbReader.IsPortable(stream))
+
+        if (File.Exists(pdbFilePath))
         {
+            using var stream = new FileHelper().GetStream(pdbFilePath, FileMode.Open, FileAccess.Read);
```

Comment:
nit: we're using `File` everywhere in this file maybe it's better use also here or create on top and reuse for future testability.

Reply:
This is just the code that was here before. Are you talking about using System.IO.File to create the stream? Or about using FileHelper to improve testability?

Reply:
I meant have all `FileHelper` or all `File` usage, but it's not mandatory I don't like the mix.

Reply:
File helper is used here to get the stream via platform abstractions. It is more correct to use FileHelper everywhere here, but that is a bigger change than what I am comfortable with without adding tests. And I don't want to give this more time right now.